REVIEW.md

```diff
@@ -22,8 +22,7 @@ generally depends on the data model and the way we talk to the database.
 Reviewers should start with the README in this folder to get a high level idea
```

Comment:
We can probably remove this whole file. This is really old and was a note to people who were reviewing the codebase as a whole.